src/overlays/actors/ovl_Bg_Hidan_Rsekizou/z_bg_hidan_rsekizou.c

```diff
@@ -214,8 +214,8 @@ Gfx* BgHidanRsekizou_DrawFireball(PlayState* play, BgHidanRsekizou* this, s16 fr
     mf->zw = (tmpf7 * coss) + this->dyna.actor.world.pos.z;
 
     gSPMatrix(displayList++,
-              Matrix_MtxFToMtx(Matrix_CheckFloats(mf, "../z_bg_hidan_rsekizou.c", 543),
-                               Graph_Alloc(play->state.gfxCtx, sizeof(Mtx))),
+              Matrix_MtxFToMtx(MATRIX_CHECK_FLOATS(mf, "../z_bg_hidan_rsekizou.c", 543),
+                               GRAPH_ALLOC(play->state.gfxCtx, sizeof(Mtx))),
```

Comment:
This is interestingly an "inlined" version of `Matrix_NewMtx`

the same thing happens in `z_bg_hidan_sekizou.c` and `z_bg_hidan_sima.c` (looks like copypasting), and... `Play_Draw`

I wonder if it was a macro, like a retail-`GRAPH_ALLOC`-like equivalent to `Matrix_NewMtx`